features/step_definitions/common.ts

```diff
@@ -37,6 +37,14 @@ Given(
   },
 );
 
+Given(
+  "Load the record numbers of the app {string} as variable: {string}",
+  function (appKey: string, destVar: string) {
+    const recordNumbers = this.getRecordNumbersByAppKey(appKey);
+    this.replacements = { [destVar]: recordNumbers, ...this.replacements };
```

Comment:
@hung-cybo 
Can we use other name for this variable, such as recordNumberList....or something like that?
It's more understandable.

Reply:
@tuanphamcybozu 

The `replacements` variable does not only store the record number list, but also other data.
So, I think we can not rename it to `recordNumberList`.